kafka-connect-s3/src/test/java/io/confluent/connect/s3/DataWriterByteArrayTest.java

```diff
@@ -150,20 +168,21 @@ protected String getDirectory(String topic, int partition) {
     return partitioner.generatePartitionedPath(topic, encodedPartition);
   }
 
-  protected List<String> getExpectedFiles(long[] validOffsets, TopicPartition tp, String extension) {
+  protected List<String> getExpectedFiles(long[] validOffsets, TopicPartition tp, String extension, CompressionType compressionType) {
```

Comment:
To methods where compression type is used only in the extension, lets avoid adding it as a parameter (and perform `extension + compressionType.extension` earlier in the caller. Sounds less painful to extend